.ci/validation.sh

```diff
@@ -738,30 +739,31 @@ no-error-strata)
 no-error-spring-integration)
   set -e
   CS_POM_VERSION="$(getCheckstylePomVersion)"
-  echo CS_version: ${CS_POM_VERSION}
+  echo CS_version: "${CS_POM_VERSION}"
   checkout_from https://github.com/spring-projects/spring-integration.git
   cd .ci-temp/spring-integration
   PROP_MAVEN_LOCAL="mavenLocal"
   PROP_CS_VERSION="checkstyleVersion"
-  ./gradlew clean check --parallel -x test -P$PROP_MAVEN_LOCAL -P$PROP_CS_VERSION=${CS_POM_VERSION}
+  ./gradlew clean check --parallel -x test -P$PROP_MAVEN_LOCAL \
+   -P$PROP_CS_VERSION="${CS_POM_VERSION}"
```

Comment:
On wrap, indentation should be +4 spaces

Reply:
Changes Done